src/components/solutions/style/solutions.css

```diff
@@ -0,0 +1,55 @@
+
+.Todo-todoList {
+    font-size:1.5rem;
+    
+}
+.Todo-todoList h2 {
+    letter-spacing: 2.5px;
+    border-bottom: 2px solid rgba(255,255,255, 0.5);
+}
+.Todo-todoList ul {
+    list-style: none;
+    margin-top:2rem ;
+}
+.Todo-todo {
```

Comment:
in the future try to nest css e.g 
.Todo-todoList .Todo-todo {.....css.....}

In this case when working on bigger application you prevent css effecting other screens, components.